### PR TITLE
finished contact page

### PR DESCRIPTION
**Pull request by reneeharder**

### CODE REVIEW

File: contact.html [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+                    </li>
+                </ul>
+            </nav>
+        </header> 
+    </div>
+  
+    <div class="container">
+        <div class="row">
+            <div class="col-sm-12 col-md-6">
+                <div class="row">
```
Comment (ayojackay) [inline contact.html:40]: this row needs to have the divs with the img tag, the doctor's name, and p tag filled with lorem to live inside. Example: 
```suggestion
                <div class="row">
                     <div class="col-xs 6 col-sm-4">
                       <img src="img/doctor1.jpg" alt= "Dr. Smith" class="-responsive">
                    </div>
                    <div class="col-sx-6 col-sm-8">
                      <h3>Dr. Joe Smith</h3>
                    </div>
                    <div class="col-xs-12 col-sm-8">
                      <p class="lorem">Lorem ipsum dolor sit amet consectetur adipisicing elit. Perferendis, nesciunt autem. Facilis minima ad libero, cum magni architecto itaque numquam facere, repudiandae quas totam sint dolore. Illum eum totam earum.</p>
                    </div>
               </div> //closes the row.     
```

File: contact.html [hunk 2]
```diff
@@ -0,0 +1,156 @@
...
+                <div class="col-xs 6 col-sm-4">
+                    <img src="img/doctor1.jpg" alt= "Dr. Smith" class="-responsive">
+                </div>
+                <div class="col-sx-6 col-sm-8">
+                    <h3>Dr. Joe Smith</h3>
+                </div>
+                <div class="col-xs-12 col-sm-8">
+                    <p class="lorem">Lorem ipsum dolor sit amet consectetur adipisicing elit. Perferendis, nesciunt autem. Facilis minima ad libero, cum magni architecto itaque numquam facere, repudiandae quas totam sint dolore. Illum eum totam earum.</p>
+                </div>
+                <div class="row">
```
Comment (ayojackay) [inline contact.html:51]: this row also needs to include the same thing for Dr. Jane Smith.

**Review by ayojackay — CHANGES_REQUESTED** [review]
Other than the fixes I am requeting from the comments, it looks great! Once you move that code where it needs to go, go ahead and add, commit, and push. then you're good to merge! Thanks.

File: contact.html (hunk 1 above)
Comment (reneeharder) [reply contact.html:40]: Okay thanks.  I just fixed it.  I appreciate your help!

**Pull request closed without merging**